webhook_to_fedora_messaging/models/service.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sqlalchemy import Column, Integer, UnicodeText, Boolean, ForeignKey
+from webhook_to_fedora_messaging.models.util import UUIDCreatableMixin, CreatableMixin
+
+from webhook_to_fedora_messaging.database import Base
+
+
+class Service(Base, UUIDCreatableMixin, CreatableMixin):
+    __tablename__ = "services"
+
+    id = Column(Integer, primary_key=True, nullable=False)
+    user_id = Column(Integer, ForeignKey("user.id", ondelete="CASCADE"), unique=False, nullable=False)
+    name = Column(UnicodeText, nullable=False)
+    type = Column(UnicodeText, nullable=False)
+    topic = Column(UnicodeText, nullable=False)
+    story = Column(UnicodeText, nullable=False)
```

Comment:
I'm not sure what this column is for. I don't see it in the ARC report either.

Reply:
I changed this to `desc`.